components/script/dom/permissions.rs

```diff
@@ -276,6 +266,23 @@ pub fn get_descriptor_permission_state(permission_name: PermissionName ,
     PermissionState::Granted
 }
 
+#[cfg(target_os = "linux")]
+fn prompt_user(permission_name: PermissionName) -> PermissionState {
+    match tinyfiledialogs::message_box_yes_no(DIALOG_TITLE,
+                                              &format!("{} {:?} ?", QUERY_DIALOG_MESSAGE, permission_name),
+                                              MessageBoxIcon::Question,
+                                              YesNo::No) {
+        YesNo::Yes => return PermissionState::Granted,
+        YesNo::No => return PermissionState::Denied,
+    };
+}
+
+#[cfg(not(target_os = "linux"))]
+fn prompt_user(_permission_name: PermissionName) -> PermissionState {
+    // TODO popup only supported on linux
+    PermissionState::Denied;
```

Comment:
Remove the `;` from the end of line. It causes the build error on windows: https://ci.appveyor.com/project/servo/servo/build/1.0.13207/job/f0pbavepdah2u611#L619